targets/TARGET_STM/TARGET_STM32L4/mbed_overrides.c

```diff
@@ -32,6 +32,10 @@ void mbed_sdk_init()
 {
     // Update the SystemCoreClock variable.
     SystemCoreClockUpdate();
```

Comment:
I think this SystemCoreClockUpdate() call (and associated comment) can be removed since this call is now being added at the end of this function, after setting up the clocks (same comment applies to all changed mbed_overrides.c files).